setup.md

```diff
@@ -0,0 +1,125 @@
+# Setup
```

Comment:
I think this setup is missing a very short snippet showing how the Riax API is used (to get the general idea, not all the details).

Also, I think that little snippet together with the "add riax as a dependency in mix" should be shown in the README. the conceptual background that you currently have in the README is ok but is less important than this.

Reply:
I've added some snippets in the readme on [this commit](https://github.com/lambdaclass/elixir_riak_core/pull/26/commits/3eb713232024fe0f55e619d7f015319ee02c178c) and an install section on [this commit](https://github.com/lambdaclass/elixir_riak_core/pull/26/commits/8bbd9b4048bca842e3d65e644397f81dce2e650f), wdy?